src/test/ui/process/process-panic-after-fork.rs

```diff
@@ -0,0 +1,151 @@
+// run-pass
+// no-prefer-dynamic
+// ignore-wasm32-bare no libc
+// ignore-windows
+// ignore-sgx no libc
+// ignore-emscripten no processes
+// ignore-sgx no processes
+
+#![feature(bench_black_box)]
+#![feature(rustc_private)]
+#![feature(never_type)]
+#![feature(panic_always_abort)]
+#![feature(test)]
```

Comment:
I think this was for `black_box` so it's not required now (since it's gated under `bench_black_box` instead).

Reply:
Let's try it and see.,.

Reply:
Hooray.  Thank you.